connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/OracleIntegrationSuite.scala

```diff
@@ -86,6 +86,8 @@ class OracleIntegrationSuite extends DockerJDBCIntegrationV2Suite with V2JDBCTes
       s"jdbc:oracle:thin:system/$oracle_password@//$ip:$port/xe"
   }
 
+  override val typeMapping: Map[DataType, DataType] = Map(StringType -> VarcharType(255))
```

Comment:
Just a quesiton. Do we need this?

Reply:
For historical reasons, we store our String as varchar through JDBC to some external systems.

Reply:
Is `255` DB-specific value? Where this `255` came from? In MySQLIntegrationSuite, it seems to be `Int.MaxValue`.

Reply:
According to the comment with it, 255 is from https://docs.oracle.com/cd/E19501-01/819-3659/gcmaz/index.html, but I think we misunderstand the doc and Int.MaxValue is much more reasonable